crates/re_viewport/src/space_view.rs

```diff
@@ -109,17 +124,22 @@ impl SpaceViewBlueprint {
         ctx: &mut ViewerContext<'_>,
         spaces_info: &SpaceInfoCollection,
         view_state: &mut dyn SpaceViewState,
+        entities_per_system: &EntitiesPerSystem,
```

Comment:
Rename to: "entities_per_system_for_class" or just pass through EntitiesPerSystemPerClass directly.